components/Molecule/graduates.html

```diff
@@ -0,0 +1,401 @@
+<style>
+html{
+	scroll-behavior: smooth;
+}
+.graduates_profile{
+    display: grid;
+}
+.cohorts a{
```

Comment:
Looks like you're creating a cohorts component, but didn't actually crate the component and put it in the pattern library.